backend/libraries/ballerina-core/codegen/python/language constructs/header.fs

```diff
@@ -13,7 +13,7 @@ module Header =
 from __future__ import annotations
 {{ String.Join("\n",
      imports
-     |> Set.map (fun (Import i) -> i)
+     |> Set.map (fun i -> $"from {i.Source} import {i.Target}")
```

Comment:
extract in a static member on Import (also, avoid doing complex things in the middle of string formatting)

Reply:
not sure I like putting this on import -> this is a very low level thing only needed for printing, I would keep as is